src/Integration/Service/ISonarQubeServiceWrapper.cs

```diff
@@ -55,5 +56,12 @@ internal interface ISonarQubeServiceWrapper
         /// <param name="connectionInformation">Required connecting information</param>
         /// <returns>All server plugins, or null on connection failure</returns>
         IEnumerable<ServerPlugin> GetPlugins(ConnectionInformation connectionInformation, CancellationToken token);
+
+        /// <summary>
+        /// Generate a <see cref="Uri"/> to the dashboard for the given project on the provided <paramref name="connectionInformation"/>.
+        /// </summary>
+        /// <param name="connectionInformation">Server connection</param>
+        /// <param name="project">Project to generate the <see cref="Uri"/> for</param>
+        Uri CreateProjectDashboardUrl(ConnectionInformation connectionInformation, ProjectInformation project);
```

Comment:
I'm adding this as a method here on `ISonarQubeServiceWrapper` because the specific URL for the project dashboard has changed between different server versions, and also might change in the future.

Currently, the latest server version has the project dashboard URL format `/overview?id=XXX` where `XXX` is the internal project ID (the number) or key (the string).
The LTS server version uses `/dashboard/index/YYY` where `YYY` is the project key.

Luckily, the current latest server version _also_ accepts the LTS URL and redirects you to the new format, meaning that for now their implementation will be the same, but this won't necessarily always be the case.